README.md

```diff
@@ -1,5 +1,78 @@
 # `hackage-cli`
 
+A command-line tool to manage
```

Comment:
It might be helpful to supply some more keywords that clarify the context of these revisions?!

```suggestion
A command-line tool to manage package metadata
```